go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -606,6 +606,17 @@ func (be *BuiltinBackupEngine) ExecuteRestore(ctx context.Context, params Restor
 // restoreFiles will copy all the files from the BackupStorage to the
 // right place.
 func (be *BuiltinBackupEngine) restoreFiles(ctx context.Context, params RestoreParams, bh backupstorage.BackupHandle, bm builtinBackupManifest) error {
+	// For optimization, we are replacing pargzip with pgzip, so newBuiltinDecompressor doesn't have to compare and print warning for every file
```

Comment:
There should be some place where we are deleting similar code?

Reply:
It is in compression.go (newbuiltindecompressor method).. since its a helper method and even though no one else is calling it but from library point of view anyone can call that method independently. So I left that code untouched.

Reply:
but doesn't that mean that we will continue to print a line for every file + one line at the beginning?
Sorry I'm not seeing how this is being fixed.

Reply:
in compressor , we print this warning and set compression engine to pgzip, if we have following condition as true
if bm.CompressionEngine == PargzipCompressor { ...
now that we set enginevalue at the caller level, we never get true for above statement at calle and hence no warning will be printed there.